homeassistant/components/demo/vacuum.py

```diff
@@ -329,6 +177,40 @@ def set_fan_speed(self, fan_speed: str, **kwargs: Any) -> None:
             self._fan_speed = fan_speed
             self.schedule_update_ha_state()
 
+    async def async_locate(self, **kwargs: Any) -> None:
+        """Locate the vacuum's position."""
+        if self.supported_features & VacuumEntityFeature.LOCATE == 0:
```

Comment:
This shouldn't be needed. The service helper will handle it.

Reply:
I suppose you mean the supported feature checks?

Reply:
Removed